{
	"document_title": "python310-djangorestframework-3.15.1-1.1 on GA media",
	"document_type": "SUSE Patch",
	"document_publisher": {
		"type": "Vendor",
		"contact_details": "security@suse.de",
		"issuing_authority": "SUSE Security Team"
	},
	"document_tracking": {
		"identification": {
			"id": "openSUSE-SU-2024:13861-1"
		},
		"status": "Final",
		"version": "1",
		"revision_history": {
			"revision": {
				"number": "1",
				"date": "2024-08-23T21:44:06Z",
				"description": "current"
			}
		},
		"initial_release_date": "2024-08-23T21:44:06Z",
		"current_release_date": "2024-08-23T21:44:06Z",
		"generator": {
			"engine": "cve-database/bin/generate-cvrf.pl",
			"date": "2017-02-24T01:00:00Z"
		}
	},
	"document_notes": [
		{
			"text": "python310-djangorestframework-3.15.1-1.1 on GA media",
			"title": "Topic",
			"type": "Summary"
		},
		{
			"text": "These are all security issues fixed in the python310-djangorestframework-3.15.1-1.1 package on the GA media of openSUSE Tumbleweed.",
			"title": "Details",
			"type": "General"
		},
		{
			"text": "The CVRF data is provided by SUSE under the Creative Commons License 4.0 with Attribution (CC-BY-4.0).",
			"title": "Terms of Use",
			"type": "Legal Disclaimer"
		},
		{
			"text": "openSUSE-Tumbleweed-2024-13861",
			"title": "Patchnames",
			"type": "Details"
		}
	],
	"documentdistribution": "Copyright SUSE LLC under the Creative Commons License 4.0 with Attribution (CC-BY-4.0)",
	"document_references": [
		{
			"type": "Self",
			"url": "https://www.suse.com/support/security/rating/",
			"description": "SUSE Security Ratings"
		},
		{
			"type": "Self",
			"url": "https://www.suse.com/security/cve/CVE-2020-25626/",
			"description": "SUSE CVE CVE-2020-25626 page"
		}
	],
	"product_tree": {
		"branch": [
			{
				"type": "Product Family",
				"name": "openSUSE Tumbleweed",
				"branch": [
					{
						"type": "Product Name",
						"name": "openSUSE Tumbleweed",
						"full_product_name": {
							"text": "openSUSE Tumbleweed",
							"product_id": "openSUSE Tumbleweed",
							"cpe": "cpe:/o:opensuse:tumbleweed"
						}
					}
				],
				"full_product_name": {}
			},
			{
				"type": "Product Version",
				"name": "python310-djangorestframework-3.15.1-1.1",
				"full_product_name": {
					"text": "python310-djangorestframework-3.15.1-1.1",
					"product_id": "python310-djangorestframework-3.15.1-1.1"
				}
			},
			{
				"type": "Product Version",
				"name": "python311-djangorestframework-3.15.1-1.1",
				"full_product_name": {
					"text": "python311-djangorestframework-3.15.1-1.1",
					"product_id": "python311-djangorestframework-3.15.1-1.1"
				}
			},
			{
				"type": "Product Version",
				"name": "python312-djangorestframework-3.15.1-1.1",
				"full_product_name": {
					"text": "python312-djangorestframework-3.15.1-1.1",
					"product_id": "python312-djangorestframework-3.15.1-1.1"
				}
			}
		],
		"relationship": [
			{
				"product_reference": "python310-djangorestframework-3.15.1-1.1",
				"relation_type": "Default Component Of",
				"relates_to_product_reference": "openSUSE Tumbleweed"
			},
			{
				"product_reference": "python311-djangorestframework-3.15.1-1.1",
				"relation_type": "Default Component Of",
				"relates_to_product_reference": "openSUSE Tumbleweed"
			},
			{
				"product_reference": "python312-djangorestframework-3.15.1-1.1",
				"relation_type": "Default Component Of",
				"relates_to_product_reference": "openSUSE Tumbleweed"
			}
		]
	},
	"vulnerability": [
		{
			"notes": [
				{
					"text": "A flaw was found in Django REST Framework versions before 3.12.0 and before 3.11.2. When using the browseable API viewer, Django REST Framework fails to properly escape certain strings that can come from user input. This allows a user who can control those strings to inject malicious <script> tags, leading to a cross-site-scripting (XSS) vulnerability.",
					"title": "Vulnerability Description",
					"type": "General"
				}
			],
			"cve": "CVE-2020-25626",
			"product_statuses": [
				{
					"type": "Fixed",
					"product_id": [
						"openSUSE Tumbleweed:python310-djangorestframework-3.15.1-1.1",
						"openSUSE Tumbleweed:python311-djangorestframework-3.15.1-1.1",
						"openSUSE Tumbleweed:python312-djangorestframework-3.15.1-1.1"
					]
				}
			],
			"threats": [
				{
					"type": "Impact",
					"description": "important"
				}
			],
			"cvss_score_sets": {
				"score_set_v_2": {
					"base_score_v_2": "4.3",
					"vector_v_2": "AV:N/AC:M/Au:N/C:N/I:P/A:N"
				},
				"score_set_v_3": {
					"base_score_v_3": "7.1",
					"vector_v_3": "CVSS:3.1/AV:N/AC:H/PR:L/UI:R/S:C/C:L/I:H/A:L"
				},
				"scoreset": {}
			},
			"remediations": [
				{
					"type": "Vendor Fix",
					"description": {
						"text": "To install this SUSE Security Update use the SUSE recommended installation methods like YaST online_update or \"zypper patch\".\n",
						"lang": "en"
					}
				}
			],
			"references": [
				{
					"url": "https://www.suse.com/security/cve/CVE-2020-25626.html",
					"description": "CVE-2020-25626"
				},
				{
					"url": "https://bugzilla.suse.com/1177205",
					"description": "SUSE Bug 1177205"
				}
			]
		}
	]
}
